src/engine/GraphStoreProtocol.h

```diff
@@ -85,6 +85,10 @@ class GraphStoreProtocol {
     updateClause::GraphUpdate up{std::move(convertedTriples), {}};
     ParsedQuery res;
     res._clause = parsedQuery::UpdateClause{std::move(up)};
+    // Graph store protocol POST requests might have a very large body. Limit
+    // the length used for the string representation (arbitrarily) to 5000.
+    string_view body = string_view(rawRequest.body()).substr(0, 5000);
+    res._originalString = absl::StrCat("Graph Store POST Operation\n", body);
```

Comment:
FYI using the first 5000 characters of the body (plus a prefix) as the representation for graph store POST operations.